src/Kunstmaan/MediaBundle/DependencyInjection/KunstmaanMediaExtension.php

```diff
@@ -16,6 +16,16 @@
  */
 class KunstmaanMediaExtension extends Extension implements PrependExtensionInterface
 {
+    const DEFAULT_CROPPING_CONFIG = [
+        'default' => [
+            ['name' => 'desktop', 'width' => 1, 'height' => 1, 'lockRation' => true],
+            ['name' => 'tablet', 'width' => 1, 'height' => 1, 'lockRatio' => true],
+            ['name' => 'phone', 'width' => 1, 'height' => 1, 'lockRatio' => true],
+        ],
+        'focus_point_classes' => ['top-left' ,'top-center', 'top-right', 'center-left', 'center', 'center-right', 'bottom-left', 'bottom-center', 'bottom-right'],
```

Comment:
Move this to a separate constant here or in the Service class. And use that constant also in the configuration default value